src/main/resources/activecollab-config/init.xml

```diff
@@ -17,30 +17,110 @@
  ~  under the License.
 -->
 <template name="init" xmlns="http://ws.apache.org/ns/synapse">
-    <parameter name="apiUrl"
-               description="The url to access the ActiveCollab API."/>
-    <parameter name="apiToken"
-               description="The api token of the account."/>
-    <parameter name="format"
-               description="The format that the reponse is expected to be."/>
+    <parameter name="apiUrl" description="The url to access the ActiveCollab API."/>
+    <parameter name="apiVersion" description="The version of the ActiveCollab API."/>
+    <parameter name="intent" description="The intent for the token."/>
+    <parameter name="clientVendor" description="The name of the instance."/>
+    <parameter name="clientName" description="The name of the client."/>
+    <parameter name="email" description="The Email ID of the user."/>
+    <parameter name="password" description="The password of an ActiveCollab account."/>
+    <parameter name="email" description="The Email ID of the user."/>
```

Comment:
Duplicate parameters. Remove it.

Reply:
@hariss63 : Has the duplicate entry been removed?

Reply:
Fixed in https://github.com/wso2-extensions/esb-connector-activecollab/pull/5